### PR TITLE
Make users get vectors without using local storage

### PR DESCRIPTION
**Pull request by godchen0212**
Make users get vectors without using local storage
Issue: #7133 

Signed-off-by: godchen <qingxiang.chen@zilliz.com>

### CODE REVIEW

**Comment by scsven** [comment]
@godchen0212 Please make the commit message more clear and meaningful or this PR will not get a "approve".

File: configs/milvus.yaml [hunk 1]
```diff
@@ -116,8 +116,9 @@ dataNode:
     clientMaxRecvSize: 104857600 # 100 MB, 100 * 1024 * 1024
     clientMaxSendSize: 104857600 # 100 MB, 100 * 1024 * 1024
 
-storage:
+localStorage:
   path: /var/lib/milvus/data/
+  enabled: true
```
Comment (xige-16) [inline configs/milvus.yaml:121]: How did you load the enable config to querynode's parameters?

**Pull request merged**